internal/pkg/service/stream/storage/repository/file/fixtures/file_open_ops_001.txt

```diff
@@ -0,0 +1,76 @@
+// READ - the source to be enabled
+➡️  TXN
+  ➡️  THEN:
+  001 ➡️  GET "definition/source/active/123/456/my-source"
+✔️  TXN | succeeded: true
+
+// READ - sinks from the source
+➡️  TXN
+  ➡️  THEN:
+  001 ➡️  GET ["definition/sink/active/123/456/my-source/", "definition/sink/active/123/456/my-source0")
+✔️  TXN | succeeded: true
+
+// READ - get statistics of the sink1 - to calculate pre-allocated disk space
+➡️  TXN
+  ➡️  THEN:
+  001 ➡️  GET ["storage/stats/staging/123/456/my-source/my-sink-1/", "storage/stats/staging/123/456/my-source/my-sink-10")
+  002 ➡️  GET ["storage/stats/target/123/456/my-source/my-sink-1/", "storage/stats/target/123/456/my-source/my-sink-10")
+✔️  TXN | succeeded: true
+
+// READ - get statistics of the sink2 - to calculate pre-allocated disk space
+➡️  TXN
+  ➡️  THEN:
+  001 ➡️  GET ["storage/stats/staging/123/456/my-source/my-sink-2/", "storage/stats/staging/123/456/my-source/my-sink-20")
+  002 ➡️  GET ["storage/stats/target/123/456/my-source/my-sink-2/", "storage/stats/target/123/456/my-source/my-sink-20")
+✔️  TXN | succeeded: true
+
+// WRITE
+➡️  TXN
+  // Objects from the READ phase must be unchanged
+  ➡️  IF:
+  001 "storage/file/all/123/456/my-source/my-sink-1/2000-01-01T03:00:00.000Z" MOD EQUAL 0
+  002 "storage/file/all/123/456/my-source/my-sink-2/2000-01-01T03:00:00.000Z" MOD EQUAL 0
+  003 "storage/slice/all/123/456/my-source/my-sink-1/2000-01-01T03:00:00.000Z/my-volume-1/2000-01-01T03:00:00.000Z" MOD EQUAL 0
+  004 "storage/slice/all/123/456/my-source/my-sink-2/2000-01-01T03:00:00.000Z/my-volume-1/2000-01-01T03:00:00.000Z" MOD EQUAL 0
+  005 "definition/source/active/123/456/my-source" MOD GREATER 0
+  006 "definition/source/active/123/456/my-source" MOD LESS %d
+  007 ["definition/sink/active/123/456/my-source/", "definition/sink/active/123/456/my-source0") MOD GREATER 0
+  008 ["definition/sink/active/123/456/my-source/", "definition/sink/active/123/456/my-source0") MOD LESS %d
```

Comment:
> Why we compare source with

- `== 0` - key exists
- `!= 0` - key doesn't exist
- `MOD < 1234(read revision)` - the key has not been changed since we read it
   - But this condition would also be fulfilled by the value `0`, therefore both conditions are necessary.
   
   Read more here:
    https://github.com/keboola/keboola-as-code/blob/331609a7b783daf22e2ba1f30b958a3bd6f2b2f9/internal/pkg/service/common/etcdop/op/atomic_do.go#L93-L157

Reply:
> Also I found suspicious the starting [ bracket and ending ) bracket

In etcd range queries, start is included, end is excluded.

We are using well known math notation:
https://www.math.utah.edu/online/1010/intervals/#:~:text=The%20notation%20may%20be%20a,it's%20said%20to%20be%20open.